### DISCUSSION TITLE
active menu module/plugin for nwg-panel

### DISCUSSION THREAD

**Discussion by aposteeltje-bwg** [nwg-piotr/nwg-shell#249] [Ideas]
Hi,
I would like to suggest a little enhancement for nwg-panel.

Instead of having each window carrying their own application-settings (file, edit, view, etc),
move that to the panel, and let the panel show that menu of the current active window., similar like the playerctl module.
Kde-plasma for example has such module, carried over from Mac-Os. (aka 'Global Menu')
See attached example screenshot.

Grt, aposteeltje
![plasma59-application-menu-bar-459057252](https://github.com/nwg-piotr/nwg-shell/assets/155484093/c94e9a60-274f-44f9-a171-c7ae081ac756)

**Comment by aposteeltje-bwg** [comment]
I mean write a module who can do ,
so users can have a choice whether to use it or not.

**Comment by nwg-piotr** [comment]
Sorry, but I've always found the global menu useless and annoying.
